src/components/VMapbox.vue

```diff
@@ -142,11 +142,6 @@ export const props = {
 
 export default {
   name: 'v-mapbox',
-  data () {
```

Comment:
We removed this before, but this caused several issues.  Please consult with Cindy.  If you don't initialize it from the data section it should be set on the object. 
You can try to set it as a "normal" property of the object. Then you can access it through `$options`, see [1]. We also tried this before but this also gave synchronisation issues. 

[1] https://vuejs.org/v2/api/#vm-options